mappings/net/minecraft/server/world/ChunkWatcher.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS tz net/minecraft/server/world/ChunkWatcher
+	CLASS tz$a Unloaded
+		FIELD a INSTANCE Ltz$a;
+	CLASS tz$b
```

Comment:
I think a good name (or at least a starting point) for that one would be `TickStatus`